py-polars/polars/internals/lazy_functions.py

```diff
@@ -1305,13 +1306,15 @@ def cumsum(
     return cumfold(lit(0).cast(UInt32), lambda a, b: a + b, column).alias("cumsum")
 
 
-def spearman_rank_corr(
-    a: str | pli.Expr, b: str | pli.Expr, ddof: int = 1, propagate_nans: bool = False
+def corr(
+    a: str | pli.Expr,
+    b: str | pli.Expr,
+    method: CorrelationMethod = "pearson",
+    ddof: int = 1,
+    propagate_nans: bool = False,
```

Comment:
`method`, `ddof` and `propogate_nans` should be keyword only imo, but I guess that is what @stinodego  is working on in another PR?

Reply:
Well, this is a new function, so let's get it right straight away. Indeed those should be keyword only.